### DISCUSSION TITLE
None of the FAQ stutter fixes FULLY fix the problem for me

### DISCUSSION THREAD

**Discussion by katimrow** [ramokz/phantom-camera#369] [Question / Help Wanted]
For context, they DO stop the stutter, they just have really weird secondary problems I cannot figure out at all. I have tried the addon (which causes weird interpolations with the character's rotation that I can't seem to figure out how to fix), the manual fix (which overrides any other change to the character's visuals), and using the follow target as the visuals node. I'm pretty stumped and not sure how to fix either of these ones to make it work. Any help would be appreciated!

**Comment by ramokz** [comment]
Would you be able to share any details or samples?

**Reply by francogp** [reply]
> Would you be able to share any details or samples?

in my case, I'm using 2d, and version 0.9 of the plugin and godot 4.4.1. My background reacts to camera position, so it check in _process the camera.get_screen_center_position(). I tried changing priorities to the node to ensure that the camera is processed first, so I know the correct camera position. But the background keep lagging behind 1 or 2 frames. If I use camera.force_update_scroll() before asking for cords, it fixes the problem, but I think that forcing an update is not a good implantation.
